divisible.rb

```diff
@@ -0,0 +1,11 @@
+def array_sort
+    my_range = Array((1..100))
+      sorted_array = Array.new
+      my_range.each { |num|
+        if num % 2 == 0 || num % 3 == 0 || num % 5 == 0
+      sorted_array.push(num)
+        end
+        } 
```

Comment:
The code gets a little hard to read when the indentation isn't consistent. I would indent it like so:

```suggestion
      my_range.each { |num|
        if num % 2 == 0 || num % 3 == 0 || num % 5 == 0
          sorted_array.push(num)
        end
      } 
```